store/mongo/datastore_mongo.go

```diff
@@ -224,7 +171,7 @@ func (db *DataStoreMongo) GetDevices(
 		findOpts.SetLimit(int64(limit))
 	}
 
-	cursor, err := collDevs.Find(ctx, fltr, findOpts)
+	cursor, err := collDevs.Find(ctx, ctxstore.WithTenantID(ctx, fltr), findOpts)
```

Comment:
I think this belongs to the parent commit.

Reply:
Done!